src/App.tsx

```diff
@@ -43,34 +43,36 @@ export default function App({ history }: AppProps) {
 
   return (
     <Router history={history}>
-      <ClientProvider>
-        <InspectorContextProvider>
-          <Sentry.ErrorBoundary fallback={errorPage}>
-            <OverlayProvider>
-              <Switch>
-                <SentryRoute exact path="/">
-                  <HomePage />
-                </SentryRoute>
-                <SentryRoute exact path="/login">
-                  <LoginPage />
-                </SentryRoute>
-                <SentryRoute exact path="/register">
-                  <RegisterPage />
-                </SentryRoute>
-                <SentryRoute path="/room/:roomId?">
-                  <RoomPage />
-                </SentryRoute>
-                <SentryRoute path="/inspector">
-                  <SequenceDiagramViewerPage />
-                </SentryRoute>
-                <SentryRoute path="*">
-                  <RoomRedirect />
-                </SentryRoute>
-              </Switch>
-            </OverlayProvider>
-          </Sentry.ErrorBoundary>
-        </InspectorContextProvider>
-      </ClientProvider>
+      <Suspense fallback={null}>
```

Comment:
Why is this necessary, do we really need to use something that is afaik experimental?

Reply:
react-i18next triggers a suspense whenever it needs to fetch a translation file, so without a `Suspense` at some level in the component tree this will crash the application. There is [a way to use it without suspense](https://react.i18next.com/latest/usetranslation-hook#not-using-suspense), but it's rather cumbersome as it involves handling the loading state separately for every single component, and suspense is already stable in React 18 (which we should update to ASAP anyways…)